### PR TITLE
Improve JSON format for data file; display projects under requirements

### PR DESCRIPTION
**Pull request by zmarvel**
As far as I can tell, this is a better structure for the projects in `data.json`. I thought it made more sense while writing these React components in `adapt.js`. Please fill me in if I'm missing something. 😸 

Like the commit messages say, these changes don't make the page do anything new besides actually displaying the groups of projects in a similar way to requirements.

### CODE REVIEW

**Comment by zmarvel** [comment]
By the way, I just used `Grid.Column`s to group each category. They seem to rearrange and resize more attractively, most of the time... see attached screenshots.
screenshot_2016-12-28_18-44-58
screenshot_2016-12-28_18-45-17
screenshot_2016-12-28_18-45-50

File: data/data.json [hunk 1]
```diff
@@ -1,255 +1,243 @@
...
     "pump": ["medtronic 512/712", "medtronic 515/715", "medtronic 522/722", "medtronic 723/523",
-     "omnipod", "t:slim", "animas ping", "animas vibe"],
-    "smartwatches": ["pebble classic", "pebble time", "android wear (any)", "apple watch", "samsung gear"],
-    "portable computers": ["raspberry pi 2", "raspberry pi 3", "intel edison"],
-    "tools/add-ons": ["explorer board", "intel base board", "carelink USB stick", "TI USB stick",
-       "cc-debugger", "WiFi adapter", "micro SD card", "2.1 amp power supply/battery"],
-    "bridges": ["xDrip", "RileyLink"],
-    "cables": ["micro-USB cable", "mini-USB cable", "USB power supply"]
-    }
+      "omnipod", "t:slim", "animas ping", "animas vibe"],
```
Comment (jessiepusateri) [inline data/data.json]: Were these extra spaces intentional?

Comment (zmarvel) [reply data/data.json]: No, I've actually fixed it on `zack-projects-2`. Been fighting with Vim today.

File: data/data.json [hunk 2]
```diff
@@ -231,13 +231,11 @@
...
     "cgm": ["dexcom g4", "dexcom g4 with share", "g5", "medtronic enlite", "medtronic revel"],
-    "pump": ["medtronic 512/712", "medtronic 515/715", "medtronic 522/722", "medtronic 723/523",
-      "omnipod", "t:slim", "animas ping", "animas vibe"],
-      "smartwatches": ["pebble classic", "pebble time", "android wear (any)", "apple watch", "samsung gear"],
-      "portable computers": ["raspberry pi 2", "raspberry pi 3", "intel edison"],
-      "tools/add-ons": ["explorer board", "intel base board", "carelink USB stick", "TI USB stick",
-        "cc-debugger", "WiFi adapter", "micro SD card", "2.1 amp power supply/battery"],
-        "bridges": ["xDrip", "RileyLink"],
-        "cables": ["micro-USB cable", "mini-USB cable", "USB power supply"]
+    "pump": ["medtronic 512/712", "medtronic 515/715", "medtronic 522/722", "medtronic 723/523", "omnipod", "t:slim", "animas ping", "animas vibe"],
```
Comment (zmarvel) [inline data/data.json:234]: @jessiepusateri here's a fix for the weird indentation.

**Comment by jessiepusateri** [comment]
Looks good to me!

**Pull request merged**